dbfit-java/postgres/src/main/java/dbfit/environment/postgres/NameNormaliserPostgres.java

```diff
@@ -0,0 +1,23 @@
+package dbfit.environment.postgres;
+
+public class NameNormaliserPostgres {
+    private NameNormaliserPostgres() {
+        // utility classes should not be instanciated
+    }
+
+
+    private static boolean isEscaped(String name) {
+    	// http://www.postgresql.org/docs/9.1/static/sql-syntax-lexical.html#SQL-SYNTAX-IDENTIFIERS
+    	return name.charAt(0) == '"' && name.charAt(name.length() - 1) == '"';
```

Comment:
This would raise `IndexOutOfBoundsException` if name is empty string - is this what we want or we rather expect `false` in such case?